Bio/Align/phylip.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2006-2016 by Peter Cock.  All rights reserved.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+"""Bio.Align support for the alignment format for input files for PHYLIP tools.
+
+You are expected to use this module via the Bio.Align functions.
+"""
+from Bio.Align import Alignment
+from Bio.Align import interfaces
+from Bio.Seq import Seq
+from Bio.SeqRecord import SeqRecord
+
+
+_PHYLIP_ID_WIDTH = 10
+
+
+class AlignmentWriter(interfaces.AlignmentWriter):
+    """Clustalw alignment writer."""
+
+    def write_alignment(self, alignment):
+        """Use this to write (another) single alignment to an open file."""
+        names = []
+        for record in alignment.sequences:
+            name = record.id.strip()
+            for char in "[](),":
+                name = name.replace(char, "")
+            for char in ":;":
+                name = name.replace(char, "|")
+            name = name[:_PHYLIP_ID_WIDTH]
+            names.append(name)
+            if "." in record.seq:
+                raise ValueError("PHYLIP format no longer allows dots in sequence")
```

Comment:
You didn't like having this multiply used message as a "constant" as per the Bio.AlignIO version of the code?

Reply:
I kept only a single check for dots in the sequence

Reply:
That sounds simpler :)